CHANGELOG.md

```diff
@@ -34,6 +34,19 @@ changes.
   + Will make bumping dependencies (e.g. cardano-node) easier.
   + Build commands change, see updated [Contribution Guidelines](https://github.com/input-output-hk/hydra/blob/master/CONTRIBUTING.md)
 
+- **BREAKING** Change the way tx validity and contestation deadline is constructed for close transactions:
+  + There is a new hydra-node flag `--contestation-period` expressed in seconds
+    to control the close tx validity bounds as well as determine the
+    contestation deadline.
```

Comment:
We should add an example: `e.g. with `--contestation-period 30s`, the node will close the head `30s` after submitting the close transaction and other parties will have another `30s` to contest.

And/or add a section in the user manual (where we describe also the fuel stuff e.g) about this.